index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <div id="iq9ue" class="row">
+      <div id="ir88y" class="cell">
+        <div id="iq53j" class="row">
+          <div id="i012r" class="cell">
+            <h1 id="title">Who are you?
+            </h1>
+          </div>
+        </div>
+        <div id="i3gfy" class="row">
+          <div id="iuewk" class="cell">
+            <p id="description">at our team we want to know more about you
+            </p>
+            <form action="" id="survey-form">
+              <div id="is1cw" class="form-group">
+                <label id="name-label" for="name">Name</label>
+                <input name="name" id="name" placeholder="enter yours here" required/>
+              </div>
+              <div class="form-group">
+                <label id="email-label" for="email">E-mail</label>
+                <input type="email" name="name" id="email" placeholder="enter yours here" required/>
+              </div>
+              <div class="form-group">
+                <label id="number-label" for="number">Phone Number</label>
+                <input type="number" min="1" max="100" name="number" id="number" placeholder="enter yours here" required/>
+              </div>
+              <div class="form-group">
+                <label for="dropdown">Options</label>
+                <select id="dropdown"><option>a</option><option>b</option></select>
```

Comment:
this is hard to read please separate options into lines to make the code more readable